### PR TITLE
Remove outdated pip_find_builddeps script

### PR DESCRIPTION
**Pull request by slimreaper35**
JIRA: STONEBLD-1333

# Maintainers will complete the following section

- [x] Commit messages are descriptive enough
- n/a Code coverage from testing does not decrease and new code is covered
- n/a Docs updated (if applicable)
- n/a Docs links in the code are still valid (if docs were updated)

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -47,7 +47,6 @@ description = black checks [Mandatory]
 skip_install = true
 commands =
     black --check --diff cachi2 tests
-    black --check --diff bin --line-length=88
```
Comment (ben-alkov) [inline tox.ini:50]: Why is this here?

Comment (slimreaper35) [reply tox.ini:50]: Exactly

**Review by brunoapimentel — APPROVED** [review]
